packages/trader/src/App/Components/Elements/PositionsDrawer/helpers/positions-helper.ts

```diff
@@ -27,7 +34,9 @@ export const getBarrierValue = (contract_info: TContractInfo) => {
     if (isDigitType(contract_info.contract_type)) {
         return digitTypeMap(contract_info)[contract_info.contract_type as keyof ReturnType<typeof digitTypeMap>];
     }
-    return addCommaToNumber(Number(contract_info.barrier));
+    return contract_info.barrier
```

Comment:
`getBarrierValue()` is used only in _packages/trader/src/App/Components/Elements/ContractAudit/contract-details.tsx_ and in tests